src/user/integrations/UserIntegrations/UserIntegrations.js

```diff
@@ -28,15 +29,31 @@ class UserIntegrations extends Component {
       integrationSelected: false,
     });
   };
-
+  handleViewSidebar = () => {
+    console.log(this.state.sideBarOpen);
+    this.setState({ sideBarOpen: !this.state.sideBarOpen });
+  };
   render() {
     const { integrationSelected, selectedIntegration } = this.state;
-
+    var sideBarClass = this.state.sideBarOpen ? "sidebar-open" : "sidebar";
     return (
       <div className="integrations">
-        <div className="navigation">
+        <div className={sideBarClass}>
           <Navigation org={this.state.org}></Navigation>
         </div>
+        <button
+          onClick={this.handleViewSidebar}
+          className="sidebar-toggle"
+          style={
+            sideBarClass === "sidebar-open"
+              ? { marginLeft: "13vw" }
+              : { marginLeft: 0 }
+          }
+        >
```

Comment:
Isn't it possible to integrate this button itself to the sidebar itself without repeating it in all the files?

Reply:
Ya, i tried implementing it with `navigation.js` file but for some reason(i am guessing related to fixed position property) the sidebar collapses but the main content doesn't resizes(failing the sole purpose of collapse feature).

Reply:
@AuraOfDivinity for your 2nd point I removed the collapsible feature for small sizes(since it was unnecessary in small screens).
Also can we not create navigation as a uniform component in layout folder?